src/main/java/sg/ncl/MainController.java

```diff
@@ -3410,4 +3408,26 @@ private String getUsageStatisticsByTeamId(String id) {
         }
         return response.getBody().toString();
     }
+
+    private String checkUserException(ExceptionState exceptionState, MyErrorResource error) {
+
+        if (exceptionState == USER_ID_NULL_OR_EMPTY_EXCEPTION || exceptionState == USER_NOT_FOUND_EXCEPTION) return error.getMessage();
+        else return null;
+
+    }
+
+    private String checkTeamException(ExceptionState exceptionState, MyErrorResource error) {
+
+        if (exceptionState == TEAM_NAME_NULL_OR_EMPTY_EXCEPTION || exceptionState == TEAM_NAME_ALREADY_EXISTS_EXCEPTION || exceptionState == TEAM_NOT_FOUND_EXCEPTION) return error.getMessage();
+        else return null;
+
+    }
+
+    private String checkDeterlabException(ExceptionState exceptionState, MyErrorResource error) {
+
+        if (exceptionState == ADAPTER_CONNECTION_EXCEPTION || exceptionState == DETERLAB_OPERATION_FAILED_EXCEPTION || exceptionState == ADAPTER_INTERNAL_ERROR_EXCEPTION) return error.getMessage();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)